datameta/api/openapi.yaml

```
@@ -1558,56 +1571,60 @@ components:
           type: boolean
         enabled:
           type: boolean
+
     MetaDataResponse:
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Identifier"
+        name:
+          type: string
+        regexDescription:
+          type: string
+          nullable: true
+        longDescription:
+          type: string
+          nullable: true
+        example:
+          type: string
+          nullable: true
+        regExp:
+          type: string
+          nullable: true
+        dateTimeFmt:
+          type: string
+          nullable: true
+        isMandatory:
+          type: boolean
+        order:
+          type: integer
+        isFile:
+          type: boolean
+        isSubmissionUnique:
+          type: boolean
+        isSiteUnique:
+          type: boolean
+        serviceId:
+          $ref: "#/components/schemas/NullableIdentifier"
+      required:
+        - id
+        - name
+        - regexDescription
+        - longDescription
+        - example
+        - regExp
+        - dateTimeFmt
+        - isMandatory
+        - order
+        - isFile
+        - isSubmissionUnique
+        - isSiteUnique
```

Looks like I forgot to add something here ;-)
```suggestion
        - isSiteUnique
        - serviceId
```